modules/migration-mtc-release-notes-1-5.adoc

```diff
@@ -37,3 +39,11 @@ This release has the following known issues:
 * PV resizing does not work as expected for AWS gp2 storage unless the `pv_resizing_threshold` is 42% or greater. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1973148[*BZ#1973148*])
 * If a migration fails, the migration plan does not retain custom PV settings for quiesced pods. You must manually roll back the migration, delete the migration plan, and create a new migration plan with your PV settings. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1784899[*BZ#1784899*])
 * Microsoft Azure storage is unavailable if you create more than 400 migration plans. The `MigStorage` custom resource displays the following message: `The request is being throttled as the limit has been reached for operation type`. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1977226[*BZ#1977226*])
+
+[id="technical-changes-1-5_{context}"]
+== Technical changes
+
+This release has the following technical changes:
+
+* The legacy {mtc-full} Operator version 1.5.1 is installed manually on {product-title} versions 3.7 to 4.5.
+* The {mtc-full} Operator version 1.5.1 is installed on {product-title} versions 4.6 to {product-version} by using the Operator Lifecycle Manager.
```

Comment:
Since this change applies to 4.6+, I was wondering if this sentence should say 4.6 and later? Just a question. Otherwise, LGTM.

Reply:
Good point! I'll change that. Thanks.